lib/p2p/Peer.ts

```diff
@@ -122,13 +108,19 @@ class Peer extends EventEmitter {
 
   public getStatus = (): string => {
     if (this.connected) {
-      return `Connected to peer (${this.id})`;
+      return `Connected to peer ${this.socketAddress.toString()}`;
```

Comment:
The public key of the node is used as identifier everywhere else. Even in the log messages. Maybe you can add it here too.

Reply:
At this point we might not know the public key of the node (at least if it's an incoming connection), this is being called before the handshake from the `initConnection` method. But you're right that it's a little unusual. Maybe it should say `Connected to peer [nodePubKey]` if it has a valid handshakeState and `Connected pre-handshake to peer [socketAddress]` otherwise?